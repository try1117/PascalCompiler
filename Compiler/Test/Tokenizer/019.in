__AydhWNC_NCB88271_128xzZZ
