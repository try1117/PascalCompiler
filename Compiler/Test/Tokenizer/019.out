0  | 0  | VARIABLE                 | __AydhWNC_NCB88271_128xzZZ| 
